upup/pkg/fi/cloud.go

```diff
@@ -169,3 +173,41 @@ func GuessCloudForZone(zone string) (CloudProviderID, bool) {
 	c, found := zonesToCloud[zone]
 	return c, found
 }
+
+// for sorting ip addresses
+type AddrSlice []address.Address
+
+func (s AddrSlice) Len() int           { return len(s) }
+func (s AddrSlice) Less(i, j int) bool { return s[i] < s[j] }
+func (s AddrSlice) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
+
+// input vpc info, output biggest free subnet(not aws subnet, bu ipv4) in vpc
```

Comment:
Let me be a bit more clear :)  Please make the func comments a sentence starting with the name of the func.